sdk/appconfiguration/app-configuration/package.json

```diff
@@ -79,7 +79,7 @@
     "tslib": "^1.10.0"
   },
   "devDependencies": {
-    "@azure/identity": "1.1.0-preview.2",
+    "@azure/identity": "^1.1.0-preview",
```

Comment:
For preview versions, the ^ does not work exactly like we would expect it to. You will need to pin the version

Reply:
Do you know if there is a way to specify use the latest preview version?

Reply:
No, we have to pin the version we need

Reply:
So the versioning pinning this way is working for preview versions. The build failed because the ai-form-recognizer commit from master had a different version. However let's do this only for dev dependencies and not actual dependencies, since actual dependencies may have breaking changes between various preview versions

Reply:
Identity is a dev dependency as well, so what are we finally going with here?

Reply:
so with the current set up, if i link to version pattern - "^1.1.0-preview" rush will link to the current local preview version. Since if you see in my PR there is no change in the pnpm lock file and in the current pnpm lock file in master, the local tgz packages are being referenced for identity. With this I won't have to update the version everytime to a new preview version